plugin/trino-hive/src/test/resources/io/trino/plugin/hive/security.json

```diff
@@ -10,7 +10,7 @@
   ],
   "schemas": [
     {
-      "owner": false
+      "owner": true
```

Comment:
Why this change?

Reply:
Otherwise, a new test in `TestHiveFileBasedSecurity` fails due to permission error. Existing tests don't depend on this condition as far as I confirmed.